manifests/server/instances.pp

```diff
@@ -0,0 +1,63 @@
+# @summary A short summary of the purpose of this defined type.
+#
+# A description of what this defined type does
+#
+# @example
+#   ssh::instances { 'namevar': }
+define ssh::server::instances (
+  String  $ensure                                        = present,
+  Hash    $options                                       = {},
+  String  $service_ensure                                = 'running',
+  Boolean $service_enable                                = true,
+  Boolean $validate_config_file                          = false,
+  Stdlib::Absolutepath $sshd_instance_config_file        = "${ssh::params::sshd_dir}/sshd_config_${title}",
+  Stdlib::Absolutepath $sshd_binary                      = $ssh::params::sshd_binary,
+  Optional[Stdlib::Absolutepath] $sshd_environments_file = $ssh::params::sshd_environments_file,
```

Comment:
Since we include `ssh` we should also rely on the parameters from the main class, not params:
```suggestion
  Stdlib::Absolutepath $sshd_instance_config_file             = "${ssh::sshd_dir}/sshd_config_${title}",
  Stdlib::Absolutepath $sshd_binary                                   = $ssh::sshd_binary,
  Optional[Stdlib::Absolutepath] $sshd_environments_file = $ssh::sshd_environments_file,
```

Reply:
not sure why GitHub renders the indent quite odd :thinking: